clash-cores.cabal

```diff
@@ -217,6 +220,7 @@ test-suite unit-tests
     tasty-quickcheck,
     tasty-th,
     hedgehog,
+    HUnit,
```

Comment:
Isn't this a mistake? I think you only use `tasty-hunit` which is already in the `build-depends`.